packages/did/src/DidDocumentExporter/README.md

```diff
@@ -1,52 +0,0 @@
-# DID Document exporter
```

Comment:
Q: Did this documentation move somewhere else?

Reply:
Was lost in one of the merges -> https://github.com/KILTprotocol/sdk-js/pull/447/commits/70f0cd2fd85255e6a289ee1f7a954d154f854208